### DISCUSSION TITLE
<title>loosed datas in Chat after synchronised Altus

### DISCUSSION THREAD

**Discussion by MartinPSA** [community/community#40656] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I have a question.  A chat history was accidentally deleted on the phone.  The complete history of the contact was loaded on the PC.  After Altus was started and then synced, it was also deleted in Altus.  Is there a way to get it back, at least on the PC?  Is it also possible to save the chat history directly to Altus for backup?  If so, where can this be found?

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/amanharwara/altus? In that case you should look for support information *in the repository for that software*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:

**Comment by jfullstackdev** [comment]
are you referring to WhatsApp chat ? because it says so in the project

![image](https://user-images.githubusercontent.com/47092464/205429653-df3010c3-f1fc-4144-8056-404973a983a8.png)

you can also ask the team, maybe they can help you too

https://github.com/WhatsApp

pick a repo, then create an issue there to ask
